tests/deprecated_api/test_remove_1-7.py

```diff
@@ -121,3 +123,26 @@ def test_v1_7_0_test_tube_logger(_, tmpdir):
 def test_v1_7_0_process_position_trainer_constructor(tmpdir):
     with pytest.deprecated_call(match=r"Setting `Trainer\(process_position=5\)` is deprecated in v1.5"):
         _ = Trainer(process_position=5)
+
+
+class BoringCallbackDDPSpawnModel(BoringModel):
+    def __init__(self):
+        super().__init__()
+
+    def add_to_queue(self, queue: torch.multiprocessing.SimpleQueue) -> None:
+        queue.put("test_val")
+        return super().add_to_queue(queue)
+
+    def get_from_queue(self, queue: torch.multiprocessing.SimpleQueue) -> None:
+        self.test_val = queue.get()
+        return super().get_from_queue(queue)
+
+
+@RunIf(min_gpus=2)
+def test_v1_7_0_deprecate_add_get_queue(tmpdir):
+    """Tests if device is set correctly when training for DDPSpawnPlugin."""
+    model = BoringCallbackDDPSpawnModel()
+    trainer = Trainer(default_root_dir=tmpdir, fast_dev_run=True, gpus=2, accelerator="ddp_spawn")
+
+    with pytest.deprecated_call(match=r"`LightningModule.add_to_queue` method was deprecated in v1.5"):
+        trainer.fit(model)
```

Comment:
```suggestion
def test_v1_7_0_deprecate_add_get_queue(tmpdir):
    """Tests if device is set correctly when training for DDPSpawnPlugin."""
    model = BoringCallbackDDPSpawnModel()
    trainer = Trainer(default_root_dir=tmpdir, fast_dev_run=True, num_processes=2, accelerator="ddp_cpu")

    with pytest.deprecated_call(match=r"`LightningModule.add_to_queue` method was deprecated in v1.5"):
        trainer.fit(model)
```
For this purpose, we can test with ddp spawn on CPU :)